week-5/F-webchat/exercise-1.js

```diff
@@ -33,5 +34,11 @@ When you open index.html in your browser, it should display the existing message
 
 */
 
+// Write your code here
 
-// Write your code here
\ No newline at end of file
+fetch("https://codeyourfuture.herokuapp.com/api/messages")
+  .then(response => response.text())
```

Comment:
should be `response.json()`

Reply:
i won't see the out-put if i make it response.json() on the message list ,,,,,it's gonna be full of this,,,,,,,,[object Object],[object Object]

Reply:
You need to parse the object array.